.github/workflows/bench.yml

```
@@ -58,7 +58,7 @@ jobs:
     # max-backjumps is increased as a temporary solution
     # for dependency resolution failure
     - if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
-      run: cabal configure --enable-benchmarks --max-backjumps 10000
+      run: cabal configure --enable-benchmarks
```

```suggestion
      run: cabal configure --enable-benchmarks --max-backjumps 12000
```

Locally it works for me without `--max-backjumps` and `--max-backjumps 12000` but no with `--max-backjumps 10000`, unfathomable mysteries of cabal solver